Makefile

```diff
@@ -6,27 +6,36 @@ ifeq ($(GALLERY_PATH),)
 GALLERY_PATH := ../../napari/examples
 endif
 
+mkfile_path := $(abspath $(lastword $(MAKEFILE_LIST)))
+current_dir := $(dir $(mkfile_path))
+docs_dir := $(current_dir)docs
+
 clean:
 	echo clean
-	rm -rf docs/_build/
-	rm -rf docs/api/napari*.rst
-	rm -rf docs/_tags
-	rm -rf docs/gallery/
+	echo $(current_dir)
+	rm -rf $(docs_dir)/_build/
+	rm -rf $(docs_dir)/api/napari*.rst
+	rm -rf $(docs_dir)/gallery/*
+	rm -rf $(docs_dir)/_tags
 
 docs-install:
-	python -m pip install -qr requirements.txt
+	python -m pip install -qr $(current_dir)requirements.txt
 
 docs-build:
-	python docs/_scripts/prep_docs.py
+	python $(docs_dir)/_scripts/prep_docs.py
 	NAPARI_APPLICATION_IPY_INTERACTIVE=0 sphinx-build -b html docs/ docs/_build -D sphinx_gallery_conf.examples_dirs=$(GALLERY_PATH)
 
+docs-xvfb:
+	python $(docs_dir)/_scripts/prep_docs.py
+	NAPARI_APPLICATION_IPY_INTERACTIVE=0 xvfb-run --auto-servernum sphinx-build -b html docs/ docs/_build -D sphinx_gallery_conf.examples_dirs=$(GALLERY_PATH)
+
```

Comment:
@psobolewskiPhD this command, if you have xvfb installed, will allow you to build the docs without seeing all of the napari windows 😍  Thanks to @dalthviz for the tip!

Reply:
I'm not sure what that is/means at the moment, but will try to look into if it's a thing on mac!

Reply:
xvfb is Linux specific.